### PR TITLE
Updated Gradle to 7.3.1, XP to 7 and plugins (#2)

### PR DESCRIPTION
**Pull request by reisfmb**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```
Comment (rymsha) [inline gradle/wrapper/gradle-wrapper.properties]: this is not enough.
Read https://docs.gradle.org/7.3.1/release-notes.html#upgrade-instructions

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,13 @@
 plugins {
-    id 'maven'
+    id 'maven-publish'
     id 'com.enonic.defaults' version '2.0.1'
     id 'com.enonic.xp.doc' version '2.0.0'
```
Comment (rymsha) [inline build.gradle]: this plugin should be removed

File: build.gradle [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 }
 
 apply plugin: 'com.enonic.xp.doc'
```
Comment (rymsha) [inline build.gradle]: if plugin is in `plugins` , this line is not needed

File: gradle.properties [hunk 4]
```diff
@@ -1,3 +1,3 @@
 group=com.enonic.guide
 projectName=guide-testing-apps
-xpVersion=6.11.0
+xpVersion=7.0.0
```
Comment (rymsha) [inline gradle.properties:3]: we update to latest XP API level (currently 7.7.0)

**Review by rymsha — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 4 above)
Comment (alansemenov) [reply gradle.properties:3]: no, we shouldn't. this is a guide, not a starter, so it should be 7.0.0

File: build.gradle [hunk 5]
```diff
@@ -1,13 +1,12 @@
 plugins {
-    id 'maven'
     id 'com.enonic.defaults' version '2.0.1'
     id 'com.enonic.xp.doc' version '2.0.0'
```
Comment (rymsha) [inline build.gradle]:
```suggestion
```

File: build.gradle [hunk 6]
```diff
@@ -1,13 +1,12 @@
...
 }
 
 apply plugin: 'com.enonic.xp.doc'
```
Comment (rymsha) [inline build.gradle]:
```suggestion
```

**Review by rymsha — CHANGES_REQUESTED** [review]

**Review by rymsha — APPROVED** [review]

**Review by alansemenov — APPROVED** [review]

**Pull request merged**